app/components/product/alias-form.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isPresent } from '@ember/utils';
+
+// https://youmightnotneed.com/lodash/#without
+// eslint-disable-next-line
+const without = (arr, ...args) => arr.filter(item => !args.includes(item))
```

Comment:
Handy helper. Maybe better to add this as a util in for example `utils/array-helpers.js`? We can probably use it in other places as well (e.g. places where we now use `splice` to remove an element from an array)